package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
:+1: 

I actually nuked this file in my PR. So I'll resolve this when I merge with your PR.